src/analysis/PointsTo/Pointer.h

```diff
@@ -63,6 +67,8 @@ struct MemoryObject
     PSNode *node;
     // possible pointers stored in this memory object
     PointsToMapT pointsTo;
+    // invalidated pointers stored in this memory object
+    InvalidatedSetT invalidated;
```

Comment:
This set is not used anywhere (as well as the functions that work with this set)